.github/workflows/test.yml

```diff
@@ -43,3 +44,95 @@ jobs:
           ATLAS_PUBLIC_KEY: ${{ secrets.ATLAS_PUBLIC_KEY }}
           ATLAS_PRIVATE_KEY: ${{ secrets.ATLAS_PRIVATE_KEY }}
 
+  prepare-e2e:
+    name: Prepare E2E configuration and image
+    needs: [unit-test, int-test]
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2.3.1
+
+    - name: Prepare tag
+      id: prepare
+      uses: ./.github/actions/set-tag
+
+    - name: Push Atlas Operator to Registry
```

Comment:
It seems, that if we push the dev image here - there's no need in the "Publish image to Registry" workflow?

Reply:
I removed push trigger from build image workflow, but I would like to leave this workflow for a while until I am sure that e2e are stable